src/one_dragon/base/push/channel/one_bot.py

```
@@ -176,4 +180,143 @@ def validate_config(self, config: dict[str, str]) -> tuple[bool, str]:
         if len(user_id) == 0 and len(group_id) == 0:
             return False, "QQ 号和群号至少需要配置一个"
 
-        return True, "配置验证通过"
\ No newline at end of file
+        return True, "配置验证通过"
+
+    # QQ 合并转发 payload 上限约 18KB（节点数 × 开销 + 总文本字符数）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)